src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let sortedPeople;
+
+  century
+    ? sortedPeople = people.filter((person) =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century
+    )
+    : sortedPeople = people.filter((person) =>
+      person.sex === 'm'
+    );
```

Comment:
Fix all cases like this:

```suggestion
     sortedPeople = century
       ? people.filter((person) =>
          person.sex === 'm' && Math.ceil(person.died / 100) === century);
        : people.filter((person) => person.sex === 'm');
```